src/bio.c

```diff
@@ -1439,6 +1439,26 @@ int wolfSSL_BIO_seek(WOLFSSL_BIO *bio, int ofs)
 
       return 0;
 }
+int wolfSSL_BIO_tell(WOLFSSL_BIO* bio)
```

Comment:
Please add a comment describing behavior of this function, thanks.

Reply:
Added a comment